.semaphore/semaphore.yml

```diff
@@ -27,6 +27,7 @@ blocks:
       jobs:
         - name: Lint && prettier
           commands:
+            - yarn add eslint-plugin-react-hooks -D -W
```

Comment:
CI complaining about not having eslint-plugin-react-hooks installed didn't dig deeper into the issue.